src/client/common/application/notebook.ts

```diff
@@ -18,7 +18,9 @@ import {
     window,
     workspace
 } from 'vscode';
+import { IS_CI_SERVER } from '../../../test/ciConstants';
```

Comment:
I don't believe that a constant like this would be an issue either way. But I've seen issues before where we referenced a check from src/test in product code and it caused an issue by causing the test code to initialize. I believe that isCI from src/client/common/constants is what we would want here. It's the same thing, but not referencing into test source from product code.

Looking at the code I see we do this in other spots, so if you want to leave this for now I can file a code-heath issue to clean up all references of this in a later checkin.

Reply:
I'll remove this

Reply:
Actually it was already removed.